internal/repository/repo.go

```diff
@@ -0,0 +1,117 @@
+package repository
+
+import (
+	model "github.com/cyneruxyz/address-book/gen/proto"
```

Comment:
Please do not make an alias for the package if you don't have a conflicts with another package names. Usually that means  you've selected wrong package name.